bin/travis/dispatch.sh

```diff
@@ -3,12 +3,11 @@
 SUITE=${1:-all}
 
 case $SUITE in
-    all )
-        bin/travis/test.sh
-        ;;
     django )
         bin/travis/test.sh
         ;;
+    djangotest )
+        bin/travis/djangocheck.sh
```

Comment:
Oh, you need to have a `;;` after this line. It is like `break` in other languages' case statements. Apparently not having it is a syntax error here (according to Travis)